pkg/orchestrator/cache.go

```diff
@@ -119,5 +126,7 @@ func incCacheMiss(nodeType NodeType) {
 		clusterHits.Add(1)
 	default:
 		log.Errorf("Cannot increment unknown nodeType, iota: %v", nodeType)
+		return
 	}
+	tlmCacheMisses.Inc(nodeType.TelemetryTags()...)
```

Comment:
how does `telemetry.NewCounter` handle if above `TelemetryTags` return `nil`?

Reply:
Ah damn great question, I'm digging into prometheus code and I think it'll panic because we are calling the short form of those methods.

I'll update the code to return `unknown, unknown` 👍 .